.eslintignore

```diff
@@ -8,3 +8,7 @@ syntest/
 .*
 *.json
 *.md
+
+
+// Since the javascript files have a lot of eslint errors that will be fixed by refactoring to typescript we ignore them for now.
```

Comment:
Should this not be a '#'? What is the comment format for ignore files?

Reply:
correct